app/controllers/rate_controller.rb

```diff
@@ -50,8 +46,14 @@ def set_rate
     @rate = Rate.find(params[:id])
```

Comment:
Ojo a asignar "rate" globalmente sin ningún tipo the scope. Esto es un problmea de seguridad.